src/Common/tests/System/Net/Http/LoopbackProxyServer.cs

```diff
@@ -206,10 +227,10 @@ private async Task<bool> ProcessRequest(StreamReader reader, StreamWriter writer
             }
         }
 
-        private async Task ProcessConnectMethod(NetworkStream clientStream, string remoteHost, int remotePort)
+        private async Task ProcessConnectMethod(Socket clientSocket, NetworkStream clientStream, string remoteHost, int remotePort)
         {
             // Open connection to destination server.
-            Socket serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            using Socket serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
```

Comment:
What does the 'using' do here?  Usually we see it in a scope with statements in braces, i.e.  

```c#
using (var socket = new Socket(...);
{
    // Use the socket.
}

// Now the socket is disposed.
```

I don't recall this single-line pattern being used before in CoreFx.  @stephentoub Can you comment on this?

Reply:
@davidsh syntax sugar introduced in C# 8; it [creates an implicit scope](https://github.com/dotnet/csharplang/blob/37efea9c833309421d9d8c35ac60b9b1c96d2037/proposals/csharp-8.0/using.md#using-declaration).

Reply:
Ok.

Personally, I think this makes the code less readable.  It's similar to why we always use braces for statement blocks in almost all cases even when it's a single statement.

This would be the first time we introduced this new syntax in the code.  I don't see the advantage of using this pattern since it makes the code harder to understand at first glance.